resources/public/js/publishing.js

```diff
@@ -7,6 +7,15 @@ var timerDelay = iosDetected ? 800 : 400;
 var $note, $action, $preview, $plain_password, $tableau;
 var backendTimer;
 
+document.addEventListener('DOMContentLoaded', function () {
+  marked.setOptions({
+    langPrefix: 'hljs lang-',
+    highlight: function (code) {
```

Comment:
Let us use a one liner here please `code => hljs.highlightAuto(code).value`. The same in `views.js`.